### DISCUSSION TITLE
Does it allow ssh git url?

### DISCUSSION THREAD

**Discussion by gator1** [mergestat/mergestat-lite#125] [Q&A]
The http(s) url such as https://github.com/gator1/terraform-provider-opentelekomcloud.git works.

The ssh url such as git@github.com:gator1/terraform-provider-opentelekomcloud.git  does not work. 
The go git url lib doesn't like `:` in the url. 

Our internal git has ssh url only. 

Thanks!

**Comment by Vialeon** [comment]
Installed by Homebrew I was able to run 
`~/$ askgit "select * from commits limit 10" --repo="git@github.com:gator1/terraform-provider-opentelekomcloud.git"`
successfully

It could be that the ssh token used is taken from your default credentials in .ssh/id_rsa.pub and .ssh/id_rsa.
If you use a different token for your internal git that might be the issue.
